core/local/watcher.js

```diff
@@ -67,11 +68,21 @@ class LocalWatcher {
     this.checksumer = checksumer.init()
   }
 
+  ensureDirSync () {
+    // This code is duplicated in local/index#start
+    if (!fs.existsSync(this.syncPath)) {
+      this.events.emit('syncdir-unlinked')
+      throw new Error('Syncdir has been unlinked')
```

Comment:
Include `syncPath` in error message?

Reply:
Error can either come in initial start throw or from a later 'syncdir-unlinked' event, which make it hard to handle them both the same.